tastyworks/dxfeed/__init__.py

```diff
@@ -1,2 +1,16 @@
 DATA_CHANNEL = '/service/data'
 SUBSCRIPTION_CHANNEL = '/service/sub'
+
+
+'''
+Random notes:
+
+com.devexperts.auth.AuthToken
+
+/meta/handshake
+/meta/connect
+/meta/unsuccessful
+/service/state
+/service/data
+/service/timeSeriesData
+'''
```

Comment:
Not sure how you feel about notes in actual files. Please let me know if there should be some sort of documentation folder to include all the information about the feeds, api, etc.

Reply:
I think notes and documentation of the tastyworks API should either be in code (with appropriate comments) or in a separate folder for documentation. Keep in mind that having "notes" in also means another thing that needs to be kept up-to-date and that is not easily testable.